scripts/exregional_get_mrms_files.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
 # This script pulls MRMS data from the NOAA HPSS
-#OBS_DIR=/scratch2/BMC/det/jwolff/HIWT/obs/mrms/proc
```

Comment:
Obviously I am guilty of included a personal path in here originally (and probably in other locations) but should we simply delete these commented out personal paths?

Reply:
Yes, sorry for not deleting!